content/json-ld/latest/events/event-bookingavailability.md

```diff
@@ -0,0 +1,221 @@
+---
+---
+
+# Event - bookingAvailability
+
+The bookingAvailability indicates whether there are still tickets or seats available for an event The following values for 'bookingAvailability' are applicable:
+* `available`: there are still tickets or seats left for the event, or the capacity for the event is unlimited.
+* `unavailable`: the event is sold out or fully booked.
+
+It is possible to set the bookingAvailability on top event level or on subEvent level:
+* **top event level**: for the whole event and all its subEvents (occurences). All subEvents will inherit the bookingAvailability from the top event level
+* **subEvent level**: for a one or multiple subEvent(s). The bookingAvailability on top event level will automtically be derived from the (different) bookingAvailability values on subEvent level.
+
+## Scope
+
+It is possible to set a bookingAvailability for the calendarTypes `single`or `multiple`. 
+It is **not** possible to set a bookingAvailability for events with calendarType `permanent` or `periodic`. 
+
+## Authorization - request headers
+
+| Header        | Value            | Required? |
+| ------------- | ---------------- | --------- |
+| Authorization | Bearer {token}   | true      |
+| X-Api-Key     | {apiKey}         | true      |
+| Content-Type  | application/json | false     |
+
+## Set status on top event level
+
+When set on top level, all subEvents of the event will inherit the bookingAvailability from the top event level.
+
+### HTTP request
+
+```
+PUT /events/{eventId}/bookingAvailability
+```
+
+Update the bookingAvailability of the top event.
+
+### Resource properties
+
+| Property	| Type | Description | Example |
+|--|--|--|--|
+| eventId	| uuid | unique identifier for an event | d595414a-13e0-4dd2-b4bd-706598427351 |
+
+### Request body
+
+| Property	| Type | Description | Example |
+|--|--|--|--|
+| bookingAvailability	| object |  |  |
+| type	| string | bookingAvailability of the event. possible vallues: `Available`, `Unavailable` | Unavailable |
+
+### Response
+
+* `204 No Content` : request successful
+* `400 Bad Request` : incorrect method, payload or URI
+* `401 Unauthorized` : expired JWT or the user behind the JWT does not have permission to perform this request
+
+### Example
+
+**Request**
+
+```
+PUT https://io-test.uitdatabank.be/events/03116768-1abc-405a-93d7-ba6ede52fe09/bookingAvailability
+Content-Type: application/json
+Authorization: Bearer {token}
+X-Api-Key: {apiKey}
+
+{
+   "type":"Unavailable",
+}
+```
+
+**Response**
+
+The following are example responses.
+
+```
+204 No Content
+```
+
+```
+400 Bad Request
+
+{
+    "validation_messages": {
+        "type": "Invalid type provided"
+    },
+    "title": "Invalid payload.",
+    "type": "about:blank"
+}
```

Comment:
Is this an actual error response that you got on this endpoint? Because this endpoint should better follow the error-response standards that we agreed on with Erwin/Luc and for example should not use `about:blank` as type.

Reply:
The error that I get for an invalid type on this endpoint is:

```
{
    "type": "https://api.publiq.be/probs/body/invalid-data",
    "title": "Invalid body data",
    "status": 400,
    "schemaErrors": [
        {
            "jsonPointer": "/type",
            "error": "The data should match one item from enum"
        }
    ]
}
```